facebook/OpenBIC
# common: sensor: Modify cpu power sensor read

Summary:
- Add **power sku unit (index 0x30)** info get for later usage which is includeing time/energy/power unit.
- Replace fixed cpu power gain **0.06103515625** with value which is calculated from power sku unit.

Test Plan:
- Build Code: PASS
- Check sensor read from BMC: PASS

Log:
- BMC console:
  ```
  root@bmc-oob:~# sensor-util mb 0x60
  mb:
  MB CPU PWR                   (0x60) :   71.00 Watts | (ok)
  ```
